src/System.Net.Sockets/src/System/Net/Sockets/Socket.cs

```diff
@@ -90,7 +90,7 @@ public Socket(AddressFamily addressFamily, SocketType socketType, ProtocolType p
             s_loggingEnabled = SocketsEventSource.Log.IsEnabled();
             if (s_loggingEnabled)
             {
-                NetEventSource.Enter(NetEventSource.ComponentType.Socket, this, "Socket", addressFamily);
+                NetEventSource.Enter(NetEventSource.ComponentType.Socket, this, nameof(Socket), addressFamily);
```

Comment:
Have we considered changing `NetEventSource.Enter` and `Exit` to use `[CallerMemberName]`?